scrapers/ak/bills.py

```diff
@@ -93,20 +98,35 @@ def scrape_bill(
         bill.add_source(url)
 
         # Get sponsors
-        spons_str = (
-            doc.xpath('//span[contains(text(), "Sponsor(S)")]')[0].getparent()[1].text
+        spons_str = "".join(
+            doc.xpath('//span[contains(text(), "Sponsor(S)")]/../strong//text()')
         ).strip()
         # Checks if there is a Sponsor string before matching
         if spons_str:
-            sponsors_match = re.match(r"(SENATOR|REPRESENTATIVE)S?", spons_str)
-            if sponsors_match:
-                sponsors = spons_str.split(",")
+            sponsors_matches = re.finditer(
+                r"(?P<sponsor_type>REPRESENTATIVE|SENATOR)S?\s+(?P<sponsors>.*)",
+                spons_str,
+            )
+
+            sponsors_cnt = 0
+
+            for sponsors_match in sponsors_matches:
+                sponsors_cnt += 1
+                sponsors_data = sponsors_match.groupdict()
+
+                sponsor_type = sponsors_data["sponsor_type"]
+                sponsors = (
+                    sponsors_data["sponsors"].replace("BY REQUEST", "").split(",")
+                )
                 sponsor = sponsors[0].strip()
 
+                sponsor_chamber = SPONSOR_CHAMBER_MAP.get(sponsor_type, "")
+
                 if sponsor:
                     bill.add_sponsorship(
-                        sponsors[0].split()[1],
+                        sponsor,
                         entity_type="person",
+                        chamber=sponsor_chamber,
                         classification="primary",
                         primary=True,
```

Comment:
Oops, one more issue I found is that sometimes the scraper says that two sponsors are primary, even though only one sponsor should be primary in those cases. What I noticed is that this happens when there are both House and Senate sponsors for a bill (upper and lower): in that case the first person in each list is marked `primary` by the scraper.

Instead, it should be: if a House bill, only first House sponsor should be primary=True, and if a Senate bill, only the first Senate sponsor should be primary=True

EG 

  - only WRIGHT should be a primary sponsor on https://www.akleg.gov/basis/Bill/Detail/33?Root=HB126
  - only KIEHL should be a primary sponsor on https://www.akleg.gov/basis/Bill/Detail/33?Root=sb10